### PR TITLE
[#227][Shaun] refactored promises in chatbot to async/await syntax.

### PR DESCRIPTION
**Pull request by ghost**
## Description
This change is non functional, we are swapping promises for async/await. This is to facilitate some error handling to come in subsequent PRs.

## Testing done
N/A

## Screenshots
N/A

## Acceptance criteria
- [ x] the chatbot still loads

### CODE REVIEW

File: src/applications/coronavirus-chatbot/index.js
Comment (shaunhubbard-va) [reply src/applications/coronavirus-chatbot/index.js:146]: this is preexisting

**Review by shefalinayak — APPROVED** [review]
lgtm

**Review by katieho-va — APPROVED** [review]

**Review by bkjohnson — APPROVED** [review]
You can [add a codeowner](https://github.com/department-of-veterans-affairs/va.gov-team/blob/46843322bbef3bfc3e4be9748601853e9af9040d/platform/engineering/codeowners.md) for `src/applications/coronavirus-chatbot` if you don't want to wait for a review from the frontend-review-group.

**Pull request merged**
